test/util/cmr.ts

```diff
@@ -0,0 +1,112 @@
+        'V1240764065-YOCLOUD', 'V1240763700-YOCLOUD', 'V1240763911-YOCLOUD',
+        'V1240763933-YOCLOUD', 'V1240763824-YOCLOUD', 'V1240764003-YOCLOUD',
+        'V1240764090-YOCLOUD', 'V1240763774-YOCLOUD', 'V1240763680-YOCLOUD',
+        'V1240763969-YOCLOUD', 'V1240764069-YOCLOUD', 'V1240763579-YOCLOUD',
+        'V1240763889-YOCLOUD', 'V1240763794-YOCLOUD', 'V1240763573-YOCLOUD',
+        'V1240763973-YOCLOUD', 'V1240764146-YOCLOUD', 'V1240763821-YOCLOUD',
+        'V1240763887-YOCLOUD', 'V1240764114-YOCLOUD', 'V1240763557-YOCLOUD',
+        'V1240764128-YOCLOUD', 'V1240763957-YOCLOUD', 'V1240763788-YOCLOUD',
+        'V1240763878-YOCLOUD', 'V1240764138-YOCLOUD', 'V1240763583-YOCLOUD',
+        'V1240764095-YOCLOUD', 'V1240763971-YOCLOUD', 'V1240764116-YOCLOUD',
+        'V1240763923-YOCLOUD', 'V1240763593-YOCLOUD', 'V1240763607-YOCLOUD',
+        'V1240764085-YOCLOUD', 'V1240763740-YOCLOUD', 'V1240763672-YOCLOUD',
+        'V1240763807-YOCLOUD', 'V1240763619-YOCLOUD', 'V1240763754-YOCLOUD',
+        'V1240764059-YOCLOUD', 'V1240763624-YOCLOUD', 'V1240764061-YOCLOUD',
+        'V1240763587-YOCLOUD', 'V1240763872-YOCLOUD', 'V1240764013-YOCLOUD',
+        'V1240764100-YOCLOUD', 'V1240763698-YOCLOUD', 'V1240763750-YOCLOUD',
+        'V1240763632-YOCLOUD', 'V1240763939-YOCLOUD', 'V1240764083-YOCLOUD',
+        'V1240763738-YOCLOUD', 'V1240763959-YOCLOUD', 'V1240763856-YOCLOUD',
+        'V1240763694-YOCLOUD', 'V1240764063-YOCLOUD', 'V1240764122-YOCLOUD',
+        'V1240763702-YOCLOUD', 'V1240763802-YOCLOUD', 'V1240763561-YOCLOUD',
+        'V1240763981-YOCLOUD', 'V1240764071-YOCLOUD', 'V1240763630-YOCLOUD',
+        'V1240763712-YOCLOUD', 'V1240764110-YOCLOUD', 'V1240763637-YOCLOUD',
+        'V1240763947-YOCLOUD', 'V1240763778-YOCLOUD', 'V1240763649-YOCLOUD',
+        'V1240763937-YOCLOUD', 'V1240763706-YOCLOUD', 'V1240763603-YOCLOUD',
+        'V1240764031-YOCLOUD', 'V1240763815-YOCLOUD', 'V1240764106-YOCLOUD',
+        'V1240763993-YOCLOUD', 'V1240763897-YOCLOUD', 'V1240763953-YOCLOUD',
+        'V1240763736-YOCLOUD', 'V1240763601-YOCLOUD', 'V1240763708-YOCLOUD',
+        'V1240763622-YOCLOUD', 'V1240763895-YOCLOUD', 'V1240764023-YOCLOUD',
+        'V1240763826-YOCLOUD', 'V1240764045-YOCLOUD', 'V1240763716-YOCLOUD',
+        'V1240763688-YOCLOUD', 'V1240763770-YOCLOUD', 'V1240763860-YOCLOUD',
+        'V1240763723-YOCLOUD', 'V1240763884-YOCLOUD', 'V1240764039-YOCLOUD',
+        'V1240764017-YOCLOUD', 'V1240763929-YOCLOUD', 'V1240764118-YOCLOUD',
+        'V1240763696-YOCLOUD', 'V1240763784-YOCLOUD', 'V1240763585-YOCLOUD',
+        'V1240764142-YOCLOUD', 'V1240763899-YOCLOUD', 'V1240764007-YOCLOUD',
+        'V1240763704-YOCLOUD', 'V1240763609-YOCLOUD', 'V1240763805-YOCLOUD',
+        'V1240763555-YOCLOUD', 'V1240763903-YOCLOUD', 'V1240763563-YOCLOUD',
+        'V1240764102-YOCLOUD', 'V1240763756-YOCLOUD', 'V1240763551-YOCLOUD',
+        'V1240763832-YOCLOUD',
+      ];
+      const variables = await getVariablesByIds(ids, '');
```

Comment:
If the request fails, would it return something other than an empty list of variables?  (I assume yes, just checking because it's not 100% obvious).  You could include one valid ID in the list to be more accurate.